samples/console_output/main.cpp

```diff
@@ -173,6 +173,26 @@ class TimeChecker : public sensr::MessageListener {
   sensr::Client* client_;
 };
 
+class ProfilingChecker : public sensr::MessageListener {
+public:
+  ProfilingChecker(sensr::Client* client) : MessageListener(ListeningType::kOutputMessage), client_(client) {}
+  void OnError(Error error, const std::string& reason) {
+    (void)reason;
```

Comment:
Why do we need to pass error here if we don't use it?

Reply:
Do you mean `reason`?
`reason` gives detail error message. but it sometime generated too often, this is the result of that.

Reply:
Ah yes, I meant `reason`. This function doesn't use `reason` so why do we need to pass it?

Reply:
Ah, this is a virtual function. Then can we add the `override` keyword?